admin/workspace-management/process-logging.md

```diff
@@ -66,9 +66,24 @@ cloud provider's log viewer, or you can use `kubectl` to print the logs:
 
 ```bash
 kubectl logs \
-  -l "com.coder.username=zac" \        # Filter by the user "zac"
-  -l "com.coder.workspace.name=code" \ # Filter by the workspace "code"
-  -c exectrace                         # Only show logs from the sidecar
+  -l "com.coder.username=admin" \        # Filter by the user "admin"
+  -l "com.coder.workspace.name=main" \ # Filter by the workspace "main"
+  -c workspace                         # Only show logs from the workspace
+```
+
+The raw logs will look something like this:
+
+```console
```

Comment:
Does our docs site have an ability to style JSON? That seems better than console for this